server/src/main/java/org/elasticsearch/index/mapper/AbstractGeometryFieldMapper.java

```diff
@@ -352,15 +373,17 @@ public QueryProcessor geometryQueryBuilder() {
     protected Explicit<Boolean> coerce;
     protected Explicit<Boolean> ignoreMalformed;
     protected Explicit<Boolean> ignoreZValue;
+    protected Explicit<Boolean> docValues;
```

Comment:
This one should be on the fieldType since it also affects queries and aggs. Only parameters that _only_ affect indexing may be stored on the mapper.

Reply:
hmmm. I suppose that was not intuitive to me. I found the relationship between the field-mapper's Explicit values, the `docValuesSet`, and the FieldType's `hasDocValues` all to compete in responsibility in some ways.

I think I see what you mean now, I will try and update to reflect this. thanks!